cookies/forms.py

```diff
@@ -33,7 +33,7 @@ class TypeModelChoiceField(forms.ModelChoiceField):
 
     def label_from_instance(self, obj):
         if obj.schema is not None:
-            return u'%s: %s' % (obj.schema.name, obj.name)
+            return u'%s (%s)' % (obj.name, obj.schema)
```

Comment:
is there a reason you use obj.schema instead of obj.schema.name?

Reply:
@prnvshrn

Reply:
I was returning the value incorrectly. Changed obj.schema to obj.schema.name